parsers/BO-enhanced.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+
+# The arrow library is used to handle datetimes consistently with other parsers
+from datetime import datetime
+from logging import Logger, getLogger
+from typing import List, NamedTuple, Optional
+
+import arrow
```

Comment:
Just providing some early feedback here but we have decided internally to move away from arrow and only use datetime when possible.